docs/victorialogs/CHANGELOG.md

```diff
@@ -18,6 +18,8 @@ according to [these docs](https://docs.victoriametrics.com/victorialogs/quicksta
 
 ## tip
 
+* FEATURE: [LogsQL](https://docs.victoriametrics.com/victorialogs/logsql/): improve [`collapse_nums` pipe](https://docs.victoriametrics.com/victorialogs/logsql/#collapse_nums-pipe) by treating `_` (underscore) as a separator for numeric tokens. This enables collapsing underscore‑delimited numbers (e.g. `temp_23_175863242537_93_98_` → `temp_<N>_<N>_<N>_<N>_`) for better normalization and grouping.
```

Comment:
Please always add links to the related issues in CHANGELOG entries ( https://github.com/VictoriaMetrics/VictoriaLogs/issues/703 in this case). This serves the following purposes:

- It allows readers investigating details by clicking the link to the related issue.
- These links are used during the release in order to properly update the linked issues with the needed information such as the release where the issue has been addressed.